src/Middleware/OAuthMiddleware.php

```diff
@@ -64,13 +64,14 @@ public function handle($request, Closure $next, $scopesString = null)
         $scopes = [];
 
         if (!is_null($scopesString)) {
-            $scopes = explode('+', $scopesString);
+
+            $scopeCombinations = explode('|', $scopesString);
```

Comment:
add a comment here that describes why we're exploding on a pipe (so that we can specify a logical "or")